packages/mobile/src/components/host-remix-contest-drawer/HostRemixContestDrawer.tsx

```diff
@@ -53,11 +56,18 @@ const mergeDateTime = (day: string, time: string) => {
 export const HostRemixContestDrawer = () => {
   const { data, onClose } = useHostRemixContestModal()
   const { mutate: createEvent } = useCreateEvent()
+  const { mutate: deleteEvent } = useDeleteEvent()
   const { mutate: updateEvent } = useUpdateEvent()
   const { data: userId } = useCurrentUserId()
   const { trackId } = data
   const { data: remixContest } = useRemixContest(trackId)
+  const { data: remixes, isLoading: remixesLoading } = useRemixes({
```

Comment:
weird q - what if there were remixes before, but then the artist started a remix competition accidentally, now they can't turn it off?

Reply:
yea thats true. we might need to update the endpoint to only grab entries from the latest contest. should be fine for now, but i'll make a ticket to discuss/fix this